### PR TITLE
Move all assets from protofire repo to glif repo and add a K8s helm charts

### PR DESCRIPTION
**Pull request by ArseniiPetrovich**

### CODE REVIEW

File: .config/init.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+docker build -t cid-checker-frontend:$(cat ./packages/frontend/version.txt) -f Dockerfile.frontend .
+docker build -t cid-checker-backend:$(cat ./packages/frontend/version.txt) -f Dockerfile.backend .
+docker build -t cid-checker-watcher:$(cat ./packages/frontend/version.txt) -f Dockerfile.watcher .
+
+echo "-----------------------------"
+echo "------ Build completed! -----"
+echo "-----------------------------"
+echo "--- Docker Compose me up! ---"
+echo "-----------------------------"
+
```
Comment (witnsby) [inline .config/init.sh:22]: the logic will execute in the CICD and I'm not sure you need to have the echo in the script.
but for a debug reason I hope you can add set +x

File: .config/init.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+docker-compose down && docker-compose up -d
+
+echo "----------------------"
+echo "--- Cleanup Docker ---"
+echo "----------------------"
```
Comment (witnsby) [inline .config/init.sh:27]: the logic will execute in the CICD and I'm not sure you need to have the echo in the script.
but for a debug reason I hope you can add set +x

File: .config/nginx.conf [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            try_files $uri $uri/ /index.html;
+        }
+
+        error_page 503 @maintenance;
+        location @maintenance {
+            rewrite ^(.*)$ /maintenance.html break;
+        }
+
+        location /api {
+            proxy_pass 'http://cid-checker-backend:3000';
```
Comment (witnsby) [inline .config/nginx.conf:60]: is it possible to make it as variable?

File: Dockerfile.backend [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+WORKDIR /app/
+
+COPY ./package.json ./package.json
+COPY ./packages/backend/config ./packages/backend/config
+COPY ./packages/backend/src ./packages/backend/src
+COPY ./packages/backend/index.js ./packages/backend/index.js
+COPY ./packages/backend/package.json ./packages/backend/package.json
+COPY ./packages/shared/package.json ./packages/shared/package.json
+COPY ./packages/shared/src ./packages/shared/src
```
Comment (witnsby) [inline Dockerfile.backend:11]: I hope you can make more beauty this piece of code
for the example 
COPY ./package.json .
COPY ./packages/backend/config .
etc

**Review by witnsby — CHANGES_REQUESTED** [review]

File: .config/init.sh (hunk 1 above)
Comment (ArseniiPetrovich) [reply .config/init.sh:22]: I'll delete it in the upcoming commit :)

File: helm/templates/cron-watcher.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (mur-me) [inline helm/templates/cron-watcher.yaml:1]: Hm, I believe this stuff is already placed in the iac repo.
See https://github.com/glifio/filecoin-iac/blob/main/k8s/cronjob_cid-checker.tf

Comment (mur-me) [reply helm/templates/cron-watcher.yaml:1]: Well, I mixed up 2 things - watcher and checker, sorry.

Comment (mur-me) [reply helm/templates/cron-watcher.yaml:1]: But we can move it into iac as future improvement

File: .config/init.sh [hunk 6]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+echo "----------------------------------------------------"
+echo "--- Init script for CID Checker (filecoin.tools) ---"
+echo "----------------------------------------------------"
+
+echo "-------------------------------------------------------"
+echo "--- Build Docker images: frontend, backend, watcher ---"
+echo "-------------------------------------------------------"
```
[the remark this answers is not in the document]
Comment (ArseniiPetrovich) [reply .config/init.sh]: Removed script

**Review by witnsby — APPROVED** [review]

**Pull request merged**